bins/packages/storcli/storcli.sh

```diff
@@ -0,0 +1,32 @@
+dependencies_storcli() {
+    apt-get update
+    apt-get install wget
+
+    echo "[+] updating soruces list"
+    wget -O - http://archive.thomas-krenn.com/tk-archive.gpg.pub | apt-key add -
+    cd /etc/apt/sources.list.d
+    wget http://archive.thomas-krenn.com/tk-main-xenial.list
+    wget http://archive.thomas-krenn.com/tk-optional-xenial.list
+    cd
+    apt-get update
+
+    echo "[+] installing storcli"
+    apt-get install storcli
```

Comment:
Again, using this way, you don't specify a version, you just use latest version they provide.